app/components/task/user/users-list-item.js

```diff
@@ -0,0 +1,47 @@
+import Ember from 'ember';
+
+const {
+  Component,
+  computed,
+  get
+} = Ember;
+
+export default Component.extend({
+  classNameBindings: ['assigned:task__user__users-list-item--assigned', 'selected:selected'],
+  classNames: ['task__user__users-list-item'],
+  tagName: ['li'],
+
+  assigned: computed('user', 'userTask', function() {
+    let user = get(this, 'user');
+    let existingUser = get(this, 'userTask.user');
+    return existingUser && get(existingUser, 'id') == get(user, 'id');
+  }),
+
+  mouseDown() {
+    let user = get(this, 'user');
+    this.sendAction('selectUser', user);
+  },
+
+  mouseEnter() {
+    let user = get(this, 'user');
+    this.sendAction('hover', user);
+  }
+
+  // selectedChanged: observer('selected', function() {
```

Comment:
We'll keep it for now commented but I do want to figure it out in the next time around.